package.json

```diff
@@ -29,14 +29,13 @@
     "coffee-script": "^1.10.0",
     "coffeeify": "^2.0.1",
     "del": "^2.1.0",
-    "event-stream": "^3.3.0",
+    "event-stream": "^3.3.4",
```

Comment:
To reduce the likelihood of someone inadvertently bumping to a newer vulnerable version, I would make this `3.3.4` rather than `^3.3.4`.